docs/container_orchestration.md

```
@@ -597,6 +597,67 @@ curl -fsSL https://get.docker.com | sh
    sudo firewall-cmd --reload
    ```
 
+#### 7. Credential File Permission Errors (VertexAI, GCP)
+
+**Symptoms:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)